### ISSUE TITLE
Case-sensitive file extensions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#20]
```
When trying to select an image, it doesn't show files with upper-case 
extensions, like .IMG
```

Original issue reported on code.google.com by `Shve...@gmail.com` on 12 Jan 2013 at 2:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 14 Jan 2013 at 12:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fix will be included tentatively in version 0.9.5
```

Original comment by `ideal2i...@gmail.com` on 14 Jan 2013 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in ver 0.9.5
```

Original comment by `ideal2i...@gmail.com` on 20 Jan 2013 at 11:56
- Changed state: **Done**
